addons/l10n_id_efaktur/views/account_move_views.xml

```diff
@@ -6,19 +6,19 @@
             <field name="model">account.move</field>
             <field name="inherit_id" ref="account.view_move_form"/>
             <field name="arch" type="xml">
-                <field name="partner_id" position="after">
+                <field name="partner_shipping_id" position="before">
                     <field name="l10n_id_need_kode_transaksi" invisible="1"/>
-                    <field name="l10n_id_attachment_id" invisible="1"/>
                     <field name="l10n_id_kode_transaksi" invisible="country_code != 'ID' or not l10n_id_need_kode_transaksi" readonly="state in ['cancel', 'posted']" required="l10n_id_need_kode_transaksi"/>
-                    <field name="l10n_id_replace_invoice_id" invisible="country_code != 'ID'" readonly="state != 'draft'" options="{'m2o_dialog': False, 'no_create': True}"/>
+                    <field name="l10n_id_efaktur_range" invisible="country_code != 'ID' or state != 'draft' or l10n_id_available_range_count &lt; 2" domain="[('available', '>', 0)]"/>
                 </field>
                 <button name="button_draft" position="after">
-                    <button name="reset_efaktur" string="Reset E-Faktur" type="object" invisible="country_code != 'ID' or not l10n_id_tax_number or state != 'cancel' or l10n_id_attachment_id"/>
+                    <button name="reset_efaktur" string="Reset E-Faktur" type="object" invisible="country_code != 'ID' or not l10n_id_tax_number or state != 'cancel' or l10n_id_efaktur_document"/>
                 </button>
                 <xpath expr=".//group[@id='other_tab_group']" position="inside">
                     <group string="Electronic Tax" invisible="country_code != 'ID'">
+                        <field name="l10n_id_replace_invoice_id" invisible="country_code != 'ID'" readonly="state != 'draft'" options="{'m2o_dialog': False, 'no_create': True}"/>
```

Comment:
Suggestion:
Doesn't matter much, but I would put this field at the bottom of the three;  it is likely that this will be used less than the other two

Reply:
```suggestion
                        <field name="l10n_id_replace_invoice_id" readonly="state != 'draft'" options="{'m2o_dialog': False, 'no_create': True}"/>
```
already set on the group